packages/blade/src/components/Collapsible/Collapsible.stories.internal.tsx

```diff
@@ -61,17 +62,38 @@ const meta: Meta<CollapsibleProps> = {
   },
 };
 
-const CollapsibleTemplate: ComponentStory<typeof CollapsibleComponent> = ({ ...args }) => {
+const CollapsibleButtonTemplate: ComponentStory<typeof CollapsibleComponent> = ({ ...args }) => {
   return (
     <CollapsibleComponent {...args}>
-      <CollapsibleButton>Answer to life, universe and everything</CollapsibleButton>
+      <CollapsibleButton>Greatest song ever</CollapsibleButton>
+      <CollapsibleBody>
+        <Text>And as we wind on down the road</Text>
+        <Text>Our shadows taller than our soul</Text>
```

Comment:
Missed opportunity to rickroll blade consumers.

Reply:
Haha, will keep in mind for future 😸